web/tests/ImJoy.spec.js

```diff
@@ -161,7 +162,20 @@ describe("ImJoy.vue", async () => {
           }
           expect(err).to.be.undefined;
           expect(data).to.equal(c);
-          done();
+          fs.writeFile("/home/test.txt", c, function(err) {
```

Comment:
Is this absolute path always safe to write to? There's no mechanism in javascript to get a safe temporary directory, like in Python?

Reply:
it should be safe  since the browser instance invoked by karma is sandboxed , so when the test initialize a browserFS, it should be clean without any existing file.

Another reason for testing directly with /tmp and /home is the backend of the two are different( in-memory storage and Indexeddb).

Reply:
But it's architecture dependent, right? Ie Linux vs Mac vs Windows.

Reply:
It independent to the OS or browser, it’s a virtual file system inside the browser, has nothing to do with the actual file system.

Reply:
Ok.